packages/graphiql-react/src/history/style.css

```diff
@@ -1,6 +1,14 @@
 .graphiql-history-header {
   font-size: var(--font-size-h2);
   font-weight: var(--font-weight-medium);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.graphiql-history-header button {
+  font-size: var(--font-size-inline-code);
+  padding: 7px 10px;
```

Comment:
Please use the `--px-` CSS variables here instead of "magic numbers"

Reply:
I'm using <HistoryContext.Provider value={history}> and implementing my own history... addToHistory, editLabel, etc all those functions.  Basically I'm storing my history in a backend instead of in localStorage and resetting the results when I click on an item in history based on the id it's stored in the backend in case results changed over time for the same query.  I played with it this morning, I think I can achieve this by removing the _id and _active like you said and then just changing the HistoryContextProvider file to pass in the whole query store item instead of individual properties (that way it will get the extra props I add on my side without having to add the extra _id and _active to this code) and then changing the HistoryStore to destructure the properties.